srml/support/src/storage/mod.rs

```diff
@@ -27,6 +27,24 @@ pub mod storage_items;
 pub mod unhashed;
 pub mod hashed;
 
+/// Execute under a transactional layer.
```

Comment:
```suggestion
/// Execute under a transactional layer.
///
```

Reply:
(if you make put space after the first line, rustdoc makes the first line a summary. )